app/players/player.py

```diff
@@ -0,0 +1,14 @@
+from abc import ABC, abstractmethod
+
+
+class Player(ABC):
+    def __init__(self, nickname: str) -> None:
+        self.nickname = nickname
+
+    @abstractmethod
+    def get_rating(self) -> None:
+        pass
+
+    @abstractmethod
+    def player_info(self) -> None:
```

Comment:
```suggestion
    def player_info(self) -> str:
```

Reply:
why would there 'Str' be if it's empty def which returns nothing? But the child class return str, and I mention it there as well.